contracts/modules/TransferManager/VolumeDumpingRestrictionManagerFactory.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.4.24;
+
+import "./VolumeDumpingRestrictionManager.sol";
+import "../ModuleFactory.sol";
+
+/**
+ * @title Factory for deploying VolumeRestrictionManager module
+ */
+contract VolumeDumpingRestrictionManagerFactory is ModuleFactory {
+
+    /**
+     * @notice Constructor
+     * @param _polyAddress Address of the polytoken
+     */
+    constructor (address _polyAddress, uint256 _setupCost, uint256 _usageCost, uint256 _subscriptionCost) public
+      ModuleFactory(_polyAddress, _setupCost, _usageCost, _subscriptionCost)
+    {
+        version = "1.0.0";
+        name = "VolumeDumpingTransferManager";
+        title = "Volume Dumping Transfer Manager";
+        description = "Manage the volume of tokens dumpable by an owner";
```

Comment:
add the compatible protocol version range. Please check the other module factories you can easily understand what I am trying to say